app/src/main/java/io/github/droidkaigi/confsched2018/util/ext/DateExt.kt

```diff
@@ -1,11 +1,25 @@
 package io.github.droidkaigi.confsched2018.util.ext
 
+import io.github.droidkaigi.confsched2018.presentation.common.pref.Prefs
 import java.text.SimpleDateFormat
 import java.util.Date
 import java.util.Locale
+import java.util.TimeZone
 
 fun Date.toReadableDateTimeString() = "${toReadableDateString()} ${toReadableTimeString()}"
 
-fun Date.toReadableDateString() = SimpleDateFormat("MM/dd", Locale.getDefault()).format(this)
+fun Date.toReadableDateString() = SimpleDateFormat("MM/dd", Locale.US).apply {
```

Comment:
🙋‍♂️  How about changing this instance to static instance to avoid creating the instance every time?

Reply:
Ah, sorry. This seems intentional according to this issue 📝 
https://github.com/DroidKaigi/conference-app-2018/pull/97